src/styles/utils/card.scss

```diff
@@ -0,0 +1,92 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: $card-width;
+  transition: transform 300ms linear;
+
+  &__image {
+    align-self: center;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: inherit;
+    line-height: 18px;
+    color: $color-main-accent;
+    margin-bottom: 4px;
+    margin-top: 40px;
+
+    &:hover {
+      color: $color-title-blue;
+      cursor: context-menu;
+    }
```

Comment:
Pay attention to this requirement
![image](https://user-images.githubusercontent.com/96409563/224280959-16f128e1-5227-4f71-b5aa-d74880cc3278.png)